Sources/TuistKit/Commands/GraphCommand.swift

```diff
@@ -9,12 +8,9 @@ import TuistSupport
 import XcodeGraph
 
 /// Command that generates and exports a dot graph from the workspace or project in the current directory.
-public struct GraphCommand: AsyncParsableCommand, HasTrackableParameters {
```

Comment:
Using service context is so much better in terms of code hygiene. I'd double-check that we are not doing `Dispatch` somewhere preventing the propagation of the container.